lib/vagrant/util/platform.rb

```diff
@@ -405,7 +406,7 @@ def wsl_to_windows_path(path)
               end
               path = [root_path, *parts].join("\\")
             else
-              path = path.to_s.sub("/mnt/", "")
+              path = path.sub("/mnt/", "")
```

Comment:
Can `path.to_s` at the bottom of this method be just `path` now with this change?

Reply:
Yep!